src/CarModel.php

```diff
@@ -25,9 +25,10 @@ public function getAllCarInfo(): array
         $cars = $query->fetchAll();
 
         $cars_ret = [];
+        $method = new CarModel(make_db());
```

Comment:
You don't need to do this, nor call getIdFromName.

Just update the query so it gets the id

"SELECT `id`. `name`, `year_made`, `zero_sixty`, `price`, `brand` FROM `cars` WHERE `deleted` = 0"

Then you can just pass $car['id'] into the new Car